src/lib.rs

```diff
@@ -3,8 +3,7 @@
 //! **mdBook** is similar to Gitbook but implemented in Rust.
 //! It offers a command line interface, but can also be used as a regular crate.
 //!
-//! This is the API doc, but you can find a [less "low-level" documentation
```

Comment:
haha yeah, I've got rustfmt set up to run every time I save. `wrap_comments = true` is set in the `rustfmt.toml` so it'll try to wrap all comments at 80 characters, which adds a bunch of noise to PRs because `lib.rs` has a few really long lines.